test/setup/iam.tf

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  int_required_roles = [
+    "roles/cloudfunctions.developer",
```

Comment:
Technically we should only include required roles here. I'd think `compute.admin` and possibly `storage.admin` is sufficient?

Reply:
Oh, I may have missed that one. I think I copied this setup from anther module. I can clean that up a bit. Stay tuned.

Reply:
This is clean now.